src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -263,7 +263,7 @@ public BigInteger(ReadOnlySpan<byte> value, bool isUnsigned = false, bool isBigE
             bool isNegative;
             if (byteCount > 0)
             {
-                byte mostSignificantByte = isBigEndian ? value[0] : value[byteCount - 1];
+                byte mostSignificantByte = isBigEndian ? value[0] : value[^1];
```

Comment:
Does this impact codegen at all? I could imagine this being equivalent to `value.Length - 1`, which might result in a secondary read from memory.

Reply:
Probably and for consistency, I'd just like to know if it is impactful at all, so we can more readily pin-point any changes in the perf issue triage.